agcaballero/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,13 +19,21 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private final ArrayList<String> mssgs = {"Hello visitor!", "I am here!", "Go away."};
```

Comment:
In general Google prefers to avoid abbreviations in variable names. I don't see this stated in Java style, but here's an example in Javascript style which I've also seen in other style guides:
"Give as descriptive a name as possible, within reason. Do not worry about saving horizontal space as it is far more important to make your code immediately understandable by a new reader. Do not use abbreviations that are ambiguous or unfamiliar to readers outside your project, and do not abbreviate by deleting letters within a word."
https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers

I'd suggest naming this variable `messages`.

Reply:
Done